Squres.py

```diff
@@ -0,0 +1,25 @@
+num1 = int(input("Enter start range:"))
+num2 = int(input("Enter end range:"))
+
+number = [*range(num1, num2, 1)]
+square = [value ** 2 for value in range(num1, num2)]
+
+database = dict(zip(number, square))
+
+counter=0
+mySet = set()
+
+while  counter<3:
```

Comment:
what if counter is greater than 3?

Reply:
if the counter is greater than 3 means you have tried the wrong input 3 times so the program will be terminated

Reply:
ok but how user know ?